db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __repr__(self) -> str:
+        return str(self.name)
```

Comment:
Consider using `__str__` instead of `__repr__` for the `Genre` model. Django typically uses `__str__` to represent objects in its admin interface and other components. This change will ensure consistency and better integration.